tests/shared/core/test_trackers.py

```diff
@@ -128,16 +133,16 @@ def test_tracker_store_storage_and_retrieval(store: TrackerStore):
     assert list(tracker.events) == [ActionExecuted(ACTION_LISTEN_NAME)]
 
     # lets log a test message
-    intent = {"name": "greet", "confidence": 1.0}
+    intent = {INTENT_NAME_KEY: "greet", "confidence": 1.0}
```

Comment:
I don't think we should use these constants in the tests, but I guess this is a broader discussion to have. 

Tests are to some extent used to make sure the API stays stable if we reuse constants we use in the code - we won't get failures if an API suddenly changes.

Reply:
Makes sense, switched back 👍